Web/Edubase.Web.UI/Assets/Scripts/GiasSearchFilters/GiasFiltering.js

```diff
@@ -360,7 +360,7 @@ class GiasFiltering {
   }
 
   showWaitingMessage(container) {
-    container.append('<div class="waiting-message"><span class="govuk-warning-text__text">Continue to wait while we load the search results.</div>');
+    container.append('<div class="waiting-message"><span class="govuk-warning-text__text">Continue to wait while the search results load.</div>');
```

Comment:
✅ This matches the current iteration of the wording/ACs.

(in fairness, so did the previous PRs at the time of submission 😁🥲 )

![image](https://github.com/user-attachments/assets/faba0e41-8169-4661-b9af-a75cb64d457a)